Tasks/Common/azurerest-common/azurerestutility.ts

```diff
@@ -157,7 +157,7 @@ function getAuthorizationToken(endPoint): Q.Promise<string> {
     var envAuthUrl = (endPoint.envAuthUrl) ? (endPoint.envAuthUrl) : defaultAuthUrl;
     var authorityUrl = envAuthUrl + endPoint.tenantID + "/oauth2/token/";
     var requestData = querystring.stringify({
-        resource: endPoint.url,
+        resource: endPoint.activeDirectoryResourceId,
```

Comment:
This change is in common utility. Can you ensure the tasks using this utility is not broken. I am assuming we have coverage in L2 for this.